dojo/tools/jfrogxray/parser.py

```diff
@@ -59,28 +58,25 @@ def get_item(vulnerability, test):
 
     cve = "No CVE on file"
     cwe = 0
+    cvssv3 = None
     cvss_v3 = "No CVSS v3 score."
     cvss_v2 = "No CVSS v2 score."
     mitigation = "N/A"
     extra_desc = ""
     # Some entries have no CVE entries, despite they exist. Example CVE-2017-1000502.
-    if 'cves' in vulnerability['component_versions']['more_details']:
-        if 'cve' in vulnerability['component_versions']['more_details']['cves'][0]:
-            cve = vulnerability['component_versions']['more_details']['cves'][0]['cve']
-
-        if 'cwe' in vulnerability['component_versions']['more_details']['cves'][0]:
-            # take only the first one for now, limitation of DD model.
-            cwe = vulnerability['component_versions']['more_details']['cves'][0]['cwe'][0].split('-')[1]
-
-            # some can be "NVD-CWE-noinfo"
-            if not type(cwe) is int:
-                cwe = 0
-
-        if 'cvss_v3' in vulnerability['component_versions']['more_details']['cves'][0]:
-            cvss_v3 = vulnerability['component_versions']['more_details']['cves'][0]['cvss_v3']
-
-        if 'cvss_v2' in vulnerability['component_versions']['more_details']['cves'][0]:
-            cvss_v2 = vulnerability['component_versions']['more_details']['cves'][0]['cvss_v2']
+    cves = vulnerability['component_versions']['more_details'].get('cves', [])
```

Comment:
This will fail if `['component_versions']['more_details']` does not exist. Maybe preserving the previous `if` statement would be good?

Reply:
The previous version fail `['component_versions']['more_details']` does not exist. I can improve that.

Reply:
Right, I read too quickly. Cheers.